test_data/sample_es7.json

```diff
@@ -0,0 +1,96 @@
+{"index":{"_index":"shakespeare","_id":2}}
+{"line_id":3,"play_name":"Henry IV","speech_number":"","line_number":"","speaker":"","text_entry":"Enter KING HENRY, LORD JOHN OF LANCASTER, the EARL of WESTMORELAND, SIR WALTER BLUNT, and others"}
+{"index":{"_index":"shakespeare","_id":3}}
```

Comment:
it was necessary to make this new file because `_type` was removed in ES7. Having documents with multiple types within one index is now not allowed and attempts to set `_type` will raise errors complaining about multiple document types (unless you explicitly include a default document type in your mapping).

It's possible this file could be irrelevant and that `es7_mapping.json` could just have some argument added to it that specifics the default document type to `line`, but I couldn't figure out how to do that in 5 minutes and this worked. Definitely would like to come back to it.

Reply:
nice, want to make an issue for that after this is merged?

Reply:
created #167